twas-base/src/main/resources/marketing-artifacts/partner-center.html

```diff
@@ -31,6 +32,8 @@ <h2>Product information links</h2>
 <ul>
 <li><a href="https://learn.microsoft.com/en-us/azure/developer/java/ee/websphere-family">WebSphere on Azure</a></li>
 <li><a href="https://ibm.biz/WebSphereOnAzureContactMe">IBM WebSphere Product Family on Azure</a></li>
+<li><a href="https://ibm.biz/twas-ND-portal">WebSphere ND Image on Azure VM</a></li>
+<li><a href="https://ibm.biz/twas-IHS-portal">WebSphere IHS Image on Azure VM</a></li>
 <li><a href="https://aka.ms/twas-single-portal">WebSphere Single Instance on Azure VM</a></li>
 <li><a href="https://aka.ms/twas-cluster-portal">WebSphere Cluster on Azure VMs</a></li>
 <li><a href="https://aka.ms/liberty-aro">WebSphere Liberty/Open Liberty on ARO</a></li>
```

Comment:
Alasdair did just created corresponding ibm.biz links for these:

```
https://ibm.biz/twas-single-portal
https://ibm.biz/twas-cluster-portal
https://ibm.biz/liberty-aro
```

Can you also please add one for 

```
<li><a href="https://ibm.biz/liberty-aks">WebSphere Liberty/Open Liberty on AKS</a></li>
```